gradle/verification-metadata.xml

```diff
@@ -750,6 +757,11 @@
             <sha256 value="9f5f3085071ec198bfc1ff6869e093fbeff988aec95624a0ca24eb048ff9eb9b" origin="Verified"/>
          </artifact>
       </component>
+      <component group="com.gradle.publish" name="plugin-publish-plugin" version="1.3.1">
```

Comment:
We can probably remove the entry above since we do not use this version of the plugin above